app/src/main/res/values/strings.xml

```diff
@@ -303,7 +303,13 @@
     <string name="sorttype_topweek">TopWeek</string>
     <string name="sorttype_topyear">TopYear</string>
     <string name="time_ago_ago">%1$s %2$s ago</string>
+    <string name="title">Title</string>
+    <string name="title_less_than_200_chars">Title cannot be > 200 chars</string>
```

Comment:
`Title cannot be more than 200 characters`

Reply:
Updated